web-console/src/utils/ingestion-spec.tsx

```diff
@@ -1206,6 +1206,30 @@ export function getIoConfigFormFields(ingestionComboType: IngestionComboType): F
             </>
           ),
         },
+        {
+          name: 'inputSource.properties.accessKeyId',
+          label: 'Access key ID',
+          type: 'string',
+          placeholder: '(optional)',
+          info: (
+            <>
+              <p>S3 access key for this S3 bucket.</p>
+              <p>Setting this will override the default configuration provided in the config.</p>
+            </>
+          ),
+        },
+        {
+          name: 'inputSource.properties.secretAccessKey',
+          label: 'Secret access key',
+          type: 'string',
```

Comment:
Druid lets you specify these fields using PasswordProvider - so you can specify an environment variable instead of pasting in your access key in the ingestion spec. Can we do this through the console?

Reply:
is there a way to specify `inputSource.properties.secretAccessKey` to be the inline thing but using a JSON object? Something like setting it to `{ "type": "inline", "key": "..." }`?

Reply:
^ I don't know exactly how this works, but @maytasm3 might...

Reply:
You can do by using the default password provider. Try `{ "type": "default", "key": "..." }`.

Reply:
If by inline you mean plain text then yes.
```
        "properties": {
            "accessKeyId": "MMMMM",
            "secretAccessKey": "mmmmm"
        },
```
is the same as 
```
        “properties”: {
            “accessKeyId”: { “type”: “default”, “variable”: “MMMMMM” },
            “secretAccessKey”: { “type”: “default”, “variable”: “mmmm” }
        },
```
The only other type of Password Provider we currently support is environment.
Example below:
```
        “properties”: {
            “accessKeyId”: { “type”: “environment”, “variable”: “ACCESS_KEY_ENV” },
            “secretAccessKey”: { “type”: “environment”, “variable”: “SECRET_KEY_ENV” }
        },
```

Reply:
But, I think this should never be an available option in the web console because of the security purpose.